.github/workflows/CI.yml

```diff
@@ -27,7 +27,7 @@ jobs:
           - 'fftw'
           - 'mkl'
         version:
-          - '1.6'
+          - 'min'
```

Comment:
I didn't realize we could do this.  So this automatically tests on the minimum-supported version from Project.toml?